_gtfobins/pg.md

```diff
@@ -0,0 +1,17 @@
+---
+functions:
+  execute-interactive:
+    - code: |
+        pg /etc/profile
+        !/bin/sh
+  file-read:
+    - code: pg file_to_read
+  sudo-enabled:
+    - code: |
+        sudo pg /etc/profile
+        !/bin/sh
+  suid-limited:
```

Comment:
Are you sure about this one?

In my case (Debian) it completely drops, but also `less`, maybe you've just copied from it. Leave this as it is, I'll fix both later.